test/UID2.Client.Test/EncryptionTestsV2.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using System;
```

Comment:
for EncryptionTestsV2/V3/V4.cs, i would suggest to change the relevant tests (not all of them) to use the new Bidstream/SharingClient as well as the legacy client. not a strong view but extra test coverage (i understand there are tests in Bidstream/SharingClientTests.cs already)

Reply:
per slack, I'm about to commit a bunch more tests